specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.1/FormRecognizer.json

```diff
@@ -1094,6 +1093,18 @@
         }
       }
     },
+    "Attributes": {
```

Comment:
Can we rename this to ModelAttributes so that this wouldn't collide with any potential future *Attributes class we may introduce?